src/haddock/modules/topology/topocg/cns/build-missing.cns

```diff
@@ -0,0 +1,296 @@
+        do (fbeta=100) (store9)
+
+        evaluate ($bath=300.0)
+        evaluate ($nstep=500)
+        evaluate ($timestep=0.0005)
+
+        do (refy=mass) (store9)
+
+        do (mass=20) (store9)
+
+        igroup interaction 
+            (store9) (store9 or known)
+        end
+
+        {- turn on initial energy terms -}
+        flags exclude * include bond angle vdw end
+ 
+        minimize powell nstep=50  nprint=10 end
+
+        set seed=$iniseed end
+
+        do (vx=maxwell($bath)) (store9)
+        do (vy=maxwell($bath)) (store9)
+        do (vz=maxwell($bath)) (store9)
+
+        flags exclude vdw include impr end
+
+        dynamics cartesian
+            nstep=50
+            timestep=$timestep
+            tcoupling=true temperature=$bath
+            nprint=$nstep
+            cmremove=false
+        end
+
+        flags include vdw end
+
+        minimize powell nstep=50 nprint=10 end
+
+        set seed=$iniseed end
+
+        do (vx=maxwell($bath)) (store9)
+        do (vy=maxwell($bath)) (store9)
+        do (vz=maxwell($bath)) (store9)
+
+        dynamics cartesian
+            nstep=50
+            timestep=$timestep
+            tcoupling=true temperature=$bath
+            nprint=$nstep
+            cmremove=false
+        end
+
+        parameter
+            nbonds
+                rcon=2. nbxmod=-3 repel=0.75
+            end
+        end
+
+        minimize powell nstep=100 nprint=25 end
+
+        set seed=$iniseed end
+
+        do (vx=maxwell($bath)) (store9)
+        do (vy=maxwell($bath)) (store9)
+        do (vz=maxwell($bath)) (store9)
+
+        dynamics cartesian
+            nstep=$nstep
+            timestep=$timestep
+            tcoupling=true temperature=$bath
+            nprint=$nstep
+            cmremove=false
+        end
+
+        {- turn on all energy terms -}
+        flags include dihe ? end
+
+        {- set repel to ~vdw radii -}
+        parameter
+            nbonds
+                repel=0.89
+            end
+        end
+
+        minimize powell nstep=500 nprint=50 end
+
+        flags exclude * include bond angl impr dihe vdw elec end
+
+        minimize powell nstep=500 nprint=50 end
+
+        {- return masses to something sensible -}
+        do (mass=refy) (store9)
+
+        set seed=$iniseed end
+
+        do (vx=maxwell($bath)) (store9)
+        do (vy=maxwell($bath)) (store9)
+        do (vz=maxwell($bath)) (store9)
+
+        dynamics cartesian
+            nstep=$nstep
+            timestep=$timestep
+            tcoupling=true temperature=$bath
+            nprint=$nstep
+            cmremove=false
+        end
+
+        {- some final minimisation -}
+        minimize powell nstep=500 drop=40.0 nprint=50 end
+
+        print thres=0.02 bonds
+        print thres=5. angles
+
+    end if
+  
+    noe reset end
+    rest dihe reset end
+
+
+    ! Check for D-amino acids
+
+    for $id in id ( name HA and not resn GLY ) loop ltod
```

Comment:
Can you describe D-amino acids in Martini? If yes this part probably needs to be checked

Reply:
I don't think we have D-form amino acid in Martini, I cannot find any information on this and considering that there is only one bead for the backbone, I think it's safe to say that there is only the L form.